### PR TITLE
BZ-1651440 Updated Applying Custom Roles and Permissions

### PR DESCRIPTION
**Pull request by lbarbeevargas**
Updated with clarification when adding new custom roles or permissions.

### CODE REVIEW

**Comment by lbarbeevargas** [comment]
BZ-1651440 https://bugzilla.redhat.com/show_bug.cgi?id=1651440

@barleyer PTAL for 3.11. The BZ has some helpful comments for context.

**Comment by lbarbeevargas** [comment]
BZ-1651440 https://bugzilla.redhat.com/show_bug.cgi?id=1651440

@openshift/team-documentation PTAL. Merge to enterprise-3.11. Add labels and milestone.

File: architecture/additional_concepts/authorization.adoc [hunk 1]
```diff
@@ -476,13 +476,15 @@ when you upgrade {product-title}.
 
 === Applying Custom Roles and Permissions
 
-To add or update custom roles and permissions, it is strongly recommended to use
+To update custom roles and permissions, it is strongly recommended to use
 the following command:
 
 ----
-# oc auth reconcile -f FILE
+# oc auth reconcile -f <file>
```
Comment (kalexand-rh) [inline architecture/additional_concepts/authorization.adoc:483]: Do we have an example of the roles file that we can link to? If so, consider adding a link to it, like:

~~~
----
# oc auth reconcile -f <file> <1>
----
<1> `<file>` is the absolute path to the xref:../../link.adoc#heading[roles and permissions] to apply.
~~~

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, but this LGTM.

File: architecture/additional_concepts/authorization.adoc (hunk 1 above)
Comment (lbarbeevargas) [reply architecture/additional_concepts/authorization.adoc]: @kalexand-rh I did some looking around to find an example of a yaml file for creating cluster roles. PTAL. Thanks!

**Comment by lbarbeevargas** [comment]
@barleyer I added a link to an example yaml file for creating cluster roles. Can you PTAL at this PR again? Thanks!

**Pull request merged**